build.gradle

```diff
@@ -27,5 +27,5 @@ allprojects {
 
 ext {
     compileSdkVersion = 29
-    hwSdkVersionName = '4.4.0'
+    hwSdkVersionName = '4.5.0'
```

Comment:
This change doesn't break any non-"internal" classes, per se, but it does add fields and features.